docs/website/tds/reference/WMS.html

```diff
@@ -25,7 +25,7 @@ <h2><a name="Which_files_can_be_served_through_the"></a>Which files can be serve
   <li>Data files must contain gridded data.</li>
   <li>The NetCDF-Java Common Data Model must be able to identify the coordinates
     system used. Check this by opening in the Grid Panel of the
-    <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">ToolsUI</a>
+    <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">ToolsUI</a>
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
Fixed